examples/Mediawiki/PageProtection.yaml

```diff
@@ -0,0 +1,170 @@
+suite: Page Protection
+description: Test page protection levels and effectiveness
+type: RPC
+
+use-fixtures:
+  - Alice
+  - RootUser
+
+prototypes:
+  edit:
+    - request:
+        method: post
+        path: api.php
+        parameters:
+          action: edit
+          format: json
+      response:
+        body:
+          edit:
+            result: Success
+
+setup:
+  # Get CSRF token for RootUser
+  - use-session: RootUser
+    request:
+      path: api.php
+      action: query
+      meta: tokens
+      format: json
+    response:
+      body:
+        query:
+          tokens:
+            csrftoken: p/grab: token
```

Comment:
As above: use Mindy the Admin (later), and let's put the edit token into the fixture. It's valid for the entire session ,we only need to ask for it once.